cellforest/utils/shell/shell_command.py

```diff
@@ -20,5 +20,19 @@ def process_shell_command(command_string, working_dir, process_name):
     logger.info(f"Finished")
 
 
+def plot_shell_command(command_string, working_dir, process_name):
+    os.makedirs(working_dir, exist_ok=True)
+    out_path = os.path.join(working_dir, "plots.out")
```

Comment:
Will each plot have it's own unique logfile, then? Or will they all be named `plots.out` and overwrite eachother if there are multiple plots for a given process?